src/main/kotlin/io/github/anki/anki/repository/mongodb/model/MongoCard.kt

```diff
@@ -0,0 +1,35 @@
+package io.github.anki.anki.repository.mongodb.model
+
+import io.github.anki.anki.controller.model.CardDto
+import io.github.anki.anki.service.model.Card
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+
+@Document(collection = "card")
+data class MongoCard(
```

Comment:
Here example of model from org api, try do somthing like this
```kotlin 
@Document(collection = MongoApiPermissionsBundle.COLLECTION_NAME)
@TypeAlias(MongoApiPermissionsBundle.TYPE_ALIAS)
internal data class MongoApiPermissionsBundle(
    @Id
    override val id: ObjectId? = null,
    @CreatedDate
    override val createdAt: Instant? = null,
    @LastModifiedDate
    override val modifiedAt: Instant? = null,

    val bundleName: String? = null,
    val authorities: List<String>? = null,
    val fieldsPermissions: MongoApiFieldsPermissions? = null
) {
  companion object {
        const val COLLECTION_NAME = "permissions_bundle"
        const val TYPE_ALIAS = "MongoApiPermissionsBundle"
    }
}
```

Reply:
+